flow/flow.vcxproj

```diff
@@ -55,6 +55,7 @@
     <ClInclude Include="CompressedInt.h" />
     <ClInclude Include="ActorCollection.h" />
     <ClInclude Include="actorcompiler.h" />
+    <ClInclude Include="Arena.cpp" />
```

Comment:
I think this might need to go up in the block above with the `ClCompile` tags.

Reply:
Done